src/arithmetic.jl

```diff
@@ -34,6 +33,145 @@ function ==(a::HomogeneousPolynomial, b::HomogeneousPolynomial)
     return iszero(a.coeffs) && iszero(b.coeffs)
 end
 
+# Iss: #209
+"""
+    <(a::Taylor1{<:Real}, b::Real)
+    <(a::TaylorN{<:Real}, b::Real)
+
+Less-than comparison of `a::Taylor1` series and `b::Real`, computed by comparing
+`constant_term(a)` and `b`. If they are equal, returns `a[nz] < 0`, with `nz` the first
+non-zero coefficient beyond the constant term. This defines a total order.
+
+Refs:
+- M. Berz, AIP Conference Proceedings 177, 275 (1988); https://doi.org/10.1063/1.37800
+- M. Berz, "Automatic Differentiation as Nonarchimedean Analysis", Computer Arithmetic and
+    Enclosure Methods, (1992), Elsevier, 439-450.
+
+---
+
+    <(a::Taylor1{<:Real}, b::Taylor1{<:Real})
+    <(a::TaylorN{<:Real}, b::Taylor1{<:Real})
+
+Return `a - b < zero(b)`.
+"""
+function <(a::Taylor1{<:Real}, b::Real)
```

Comment:
I would strongly advise checking for method ambiguities introduced by this change using Aqua.jl.

Reply:
I've checked it, and it shows that there are `unbound_args` and `stale_deps` (`IntervalArithmetic`, which is rather a conditional dependency). So no method ambiguities! Thanks for the suggestion!